apps/web/modules/survey/templates/components/template-container.tsx

```diff
@@ -18,13 +18,15 @@ type TemplateContainerWithPreviewProps = {
   environment: Pick<Environment, "id" | "appSetupCompleted">;
   userId: string;
   prefilledFilters: (TProjectConfigChannel | TProjectConfigIndustry | TTemplateRole | null)[];
+  isTemplatePage: boolean;
```

Comment:
I think we can also pass a default value here as `true` since this is primarily used in the templates components, we won't have to pass it as true then from the templates page